cico_run_tests.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
+# This file is supposed to be executed by CICO build (ci.centos.org/view/Devtools/)
+#   1. Run tests for fabric8-planner
+#   2. Create snapshot with fabric8-ui's master and fabric8-planner's current changes
+
+# Show command before executing
+set -x
+
+# Exit on error
+set -e
+
+# Source environment variables of the jenkins slave
+# that might interest this worker.
+if [ -e "jenkins-env" ]; then
+  cat jenkins-env \
+    | grep -E "(JENKINS_URL|DEVSHIFT_USERNAME|DEVSHIFT_PASSWORD|GIT_BRANCH|GIT_COMMIT|BUILD_NUMBER|ghprbSourceBranch|ghprbActualCommit|BUILD_URL|ghprbPullId)=" \
+    | sed 's/^/export /g' \
+    > /tmp/jenkins-env
+  source /tmp/jenkins-env
+fi
+
+# We need to disable selinux for now, XXX
+/usr/sbin/setenforce 0
+
+# Get gcc-c++ to avoid npm errors
+yum -y install gcc-c++
+
+# Get all the deps in
+yum -y install docker make git
+service docker start
+
+# Build builder image
+cp /tmp/jenkins-env .
+
+mkdir -p fabric8-ui-dist
+# Build fabric8-planner image
+docker build -t fabric8-planner-builder -f Dockerfile .
```

Comment:
`-f Dockerfile` is default and can be safely ignored.
`-f` is only needed to be mentioned for Dockerfiles with different names.

Reply:
thanks done in https://github.com/fabric8-ui/fabric8-planner/pull/2473/commits/537acb7716f3dd17ce1e0ae13a669e45edb24da0